app/controllers/ats/positions_controller.rb

```diff
@@ -321,44 +321,42 @@ def position_status_options(position)
   end
 
   def set_side_header_predefined_options
-    @options_for_collaborators =
+    members_query =
       Member
       .includes(:account)
+      .order("accounts.name")
+
+    @options_for_collaborators =
+      members_query
       .where.not(id: @position.recruiter_id)
       .where(access_level: Position::COLLABORATORS_ACCESS_LEVEL)
       .map do |member|
-        {
-          text: member.account.name,
-          value: member.id,
-          selected: @position.collaborator_ids&.include?(member.id)
-        }
+        predefined_option(member, :collaborator_ids)
       end
 
     @options_for_hiring_managers =
-      Member
-      .includes(:account)
+      members_query
       .where(access_level: Position::HIRING_MANAGERS_ACCESS_LEVEL)
       .map do |member|
-        {
-          text: member.account.name,
-          value: member.id,
-          selected: @position.hiring_manager_ids&.include?(member.id)
-        }
+        predefined_option(member, :hiring_manager_ids)
       end
 
     @options_for_interviewers =
-      Member
-      .includes(:account)
+      members_query
       .where(access_level: Position::INTERVIEWERS_ACCESS_LEVEL)
       .map do |member|
-        {
-          text: member.account.name,
-          value: member.id,
-          selected: @position.interviewer_ids&.include?(member.id)
-        }
+        predefined_option(member, :interviewer_ids)
       end
   end
 
+  def predefined_option(member, field_name)
```

Comment:
Perhaps you would like to suggest a better name for the method?

Reply:
Maybe 
```suggestion
  def compose_options_for_select(member, field_name)
```
[reference1](https://github.com/freeats/freeats/blob/c39fe676ff551a9aa6e3dd13804a537e6c796b82/app/helpers/ats/members_helper.rb#L4), [reference2](https://github.com/freeats/freeats/blob/c39fe676ff551a9aa6e3dd13804a537e6c796b82/app/helpers/application_helper.rb#L69C7-L69C8)

What do you think?